core/src/main/java/com/graphhopper/routing/util/parsers/helpers/OSMValueExtractor.java

```diff
@@ -28,23 +24,23 @@ private OSMValueExtractor() {
     }
 
     public static void extractTons(IntsRef edgeFlags, ReaderWay way, DecimalEncodedValue valueEncoder, List<String> keys, boolean enableLog) {
```

Comment:
Would you also remove this enableLog parameter :) ? (Same also for extractMeter)

Reply:
Ah, thanks. Missed it!